packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -40,13 +58,36 @@ export const CloudPulseRegionSelect = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [regions]);
 
+    // validate launchDrakly region_ids with the ids from the fetched 'all-regions'
+    const supportedRegions = React.useMemo<Region[] | undefined>(() => {
+      const resourceTypeFlag:
+        | CloudPulseResourceTypeMapFlag
+        | undefined = flags.aclpResourceTypeMap?.find(
+        (item: CloudPulseResourceTypeMapFlag) =>
+          item.serviceType === serviceType
+      );
```

Comment:
```suggestion
      const resourceTypeFlag = flags.aclpResourceTypeMap?.find(
        (item: CloudPulseResourceTypeMapFlag) =>
          item.serviceType === serviceType
      );
```

No need to provide types, as typescript will automatically detect it